core/chains/evm/types/nonce.go

```diff
@@ -17,3 +17,7 @@ func (n Nonce) Int64() int64 {
 func (n Nonce) String() string {
 	return strconv.FormatInt(n.Int64(), 10)
 }
+
+func NextNonceFunc(last Nonce) Nonce {
```

Comment:
This would instead be:

func (n Nonce) Next() Nonce {
	return Nonce(n + 1)
}

Reply:
Where is this used? I think it would be fine to just inline the definition instead of declaring a separate func.

Reply:
The next sequence function or method would only be used [here](https://github.com/smartcontractkit/chainlink/blob/f252c1f1acb532f630fb83216a1f1ef5ebce228e/common/txmgr/broadcaster.go#L349) and [here](https://github.com/smartcontractkit/chainlink/blob/f252c1f1acb532f630fb83216a1f1ef5ebce228e/common/txmgr/broadcaster.go#L831) in the broadcaster